internal/app/go-filecoin/node/block_propagate_test.go

```diff
@@ -154,8 +153,6 @@ func makeNodesBlockPropTests(t *testing.T, numNodes int) (address.Address, []*No
 	minerNode := builder.Build(ctx)
 	seed.GiveKey(t, minerNode, 0)
 	mineraddr, _ := seed.GiveMiner(t, minerNode, 0)
-	_, err := storage.NewMiner()
```

Comment:
unskip this test per comment 106!

Reply:
Also unskip `TestBlockPropsManyNodes` neither of these is expected to fail anymore.

Reply:
These tests depend on new storage miner actor functionality that isn't currently integrated (i.e. they currently fail because we fail to retrieve valid SectorInfo structs from the miner actor). I changed the skip string to note this.